core/src/types/ops.rs

```diff
@@ -320,12 +320,10 @@ impl OpStat {
 /// Args for `write` operation.
 #[derive(Debug, Clone, Default)]
 pub struct OpWrite {
-    append: bool,
```

Comment:
If `write` only does like `append`, isn't that a bit different from op's `write` and confusing?